Doppler.HtmlEditorApi/Domain/CampaignState.cs

```diff
@@ -35,4 +35,7 @@ public record TestABCampaignState(
     int? EditorType,
     CampaignStatus?
     CampaignStatus,
-    TestABCondition TestABCondition) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition);
+    TestABCondition TestABCondition,
+    int? IdCampaignA,
+    int? IdCampaignB,
+    int? IdCampaignResult) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition);
```

Comment:
Es discutible, pero yo agregaría estos campos al CampaignState abstracto para evitar casteos:

```suggestion
    int? IdCampaignA,
    int? IdCampaignB,
    int? IdCampaignResult) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition, IdCampaignA, IdCampaignB, IdCampaignResult);
```

Reply:
Osea, los tipos ClassicCampaignState y TestABCampaignState solo servirían para tener los constructres con nombre.

Reply:
Done 👍